juntagrico/templates/createsubscription/select_subscription.html

```diff
@@ -3,7 +3,7 @@
 {% load config %}
 {% block all_page_title %}
     {% vocabulary "subscription" as v_subscription %}
-    <div class="offset-md-3 col-md-9 juntagrico">
+    <div class="offset-md-2 col-md-8 juntagrico">
```

Comment:
why? its a title isnt it?

Reply:
```suggestion
    <div class="offset-md-3 col-md-9 juntagrico">
```
Can be reverted here. #83 changes this consistently.